sdk/common/src/main/java/io/opentelemetry/sdk/common/InstrumentationScopeInfo.java

```diff
@@ -30,7 +30,12 @@ public static InstrumentationScopeInfo create(String name) {
     return InstrumentationScopeInfo.create(name, null, null, Attributes.empty());
   }
 
-  /** Creates a new instance of {@link InstrumentationScopeInfo}. */
+  /**
+   * Creates a new instance of {@link InstrumentationScopeInfo}.
+   *
+   * @deprecated Use {@link #builder(String)}.
```

Comment:
```suggestion
   * @Deprecated Use {@link #builder(String)} or {@link #create(String)}.
```

Reply:
Lowercase `@deprecated` is the [correct](https://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html) tag for javadoc. But will add reference to create method.

Reply:
Wow, thanks, I learned something about javadoc that I now wish I didn't know. 😁